### ISSUE TITLE
Add WebAssembly builds to releases section.

### ISSUE THREAD

**Issue by oofdere** [libjxl/libjxl#2200]
**Is your feature request related to a problem? Please describe.**
There are no readily-available Wasm builds of libjxl available except for [the really old one in Squoosh](https://github.com/GoogleChromeLabs/squoosh/tree/dev/codecs/jxl), last updated over a year ago!

**Describe the solution you'd like**
Wasm builds are already [built and tested](https://github.com/libjxl/libjxl/blob/main/.github/workflows/build_test.yml#L348) through GitHub Actions, but are not added to the releases, so I'd like for them to be added.

**Describe alternatives you've considered**
I'm currently using the aformentioned version in Squoosh but it's extremely out of date, so much worse performance and probably also security concerns with that.

**Comment by fredprodibi** [comment]
I agree, if you want to ease adoption. I think you should add at least two wasm build.
- one as slim as possible with decode only features
- one with encode / decode

Also if progressive decoding is really something you would like to push It may also be part of the decoder. May be via html canvas or something like that.

**Comment by oofdere** [comment]
> I agree, if you want to ease adoption. I think you should add at least two wasm build.
> 
>     * one as slim as possible with decode only features
> 
>     * one with encode / decode

AFAIK the Wasm builds should basically be the distinct `cjxl` and `djxl` programs, as mentioned in https://github.com/libjxl/libjxl/issues/1165#issuecomment-1032231249

> Also if progressive decoding is really something you would like to push It may also be part of the decoder. May be via html canvas or something like that.

Progressive decoding might be a bit much to ask for at this stage of development, but would be cool for sure. The HTML/Canvas stuff would presumably be out of scope for this project though.
